### PR TITLE
fix nbconvert version

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -23,4 +23,4 @@ pyyaml
 scrapbook
 matplotlib
 pytest
-werkzeug==2.0.3
```
Comment (volodymyrss) [inline requirements.txt:26]: so what's the change?

**Comment by burnout87** [comment]
Probably the problem comes from the `nbconvert` version. Testing ongoing

**Comment by burnout87** [comment]
Perhaprs we should upgrade the version and push it?

**Comment by dsavchenko** [comment]
Could `>=7.0.0` be used instead of fixing the version?

**Comment by burnout87** [comment]
> Could `>=7.0.0` be used instead of fixing the version?

I have no objection so I will try it

**Comment by burnout87** [comment]
> Could `>=7.0.0` be used instead of fixing the version?

Just tried and I confirm it works

**Review by dsavchenko — APPROVED** [review]

**Pull request merged**